### DISCUSSION TITLE
macOSでの実行方法

### DISCUSSION THREAD

**Discussion by ndsuzuki** [latteart-org/latteart#17] [Q&A] [answered]
macOS上でのツールの起動方法を教えていただけないでしょうか？  
提供されているツール起動用のvbsファイルは、macOSでは実行できないため質問させていただきました。

**Comment by knukio** [comment, marked as answer]
現状は自分でソースコードからビルドする必要がありますが、今後のリリースではmacOS版のバイナリもアップロードできるようにしたいと思います。
また、v1.7.5のMac版バイナリも準備でき次第アップロードしますのでお待ちください。

**Reply by knukio** [reply]
v1.7.5 と v2.0.0のMac版バイナリをアップロードしました。
現状、Macでの利用は実験的機能としています。

利用方法は[リンク](https://github.com/latteart-org/latteart/blob/main/docs/startup/pc.md#mac-%E3%81%A7%E3%81%AE%E5%88%A9%E7%94%A8%E5%AE%9F%E9%A8%93%E7%9A%84)をご参照ください。

**Reply by ndsuzuki** [reply]
提供ありがとうございます。  
早速、利用させていただきます。
